quipucords/tests/api/credential/test_credential.py

```diff
@@ -18,14 +16,23 @@
 from tests.factories import CredentialFactory, SourceFactory
 from tests.mixins import LoggedUserMixin
 
+ACCEPT_JSON_HEADER = {"Accept": "application/json"}
 
-class CredentialTest(LoggedUserMixin, TestCase):
+
+def alt_cred_type(cred_type):
+    """Given a credential type, return an alternate random one."""
+    cred_types = DataSources.values.copy()
+    cred_types.remove(cred_type)
+    return random.choice(cred_types)
+
+
+@pytest.mark.django_db
+class TestCredential(LoggedUserMixin):
     """Tests against the Credential model and view set."""
 
-    def setUp(self):
+    def setup_method(self, _test_method):
         """Create test case setup."""
         management.call_command("flush", "--no-input")
```

Comment:
now that this test is not a unittest anymore I think we can skip this setup.

pytest-django tests already teardown data between each test.

Reply:
Without this, the following 2 tests fail:

```
test_hostcred_update_view
test_hostcred_update_double
```

They assume id 2 for the update, which is true if things are torned down, but they are not somehow.  I will update the tests to use the response's id to be more reliable. But there may be something else not setup right.